ksqldb-engine/src/main/java/io/confluent/ksql/physical/pull/PullPhysicalPlan.java

```diff
@@ -182,8 +181,7 @@ public QueryId getQueryId() {
    */
   public enum PullSourceType {
     NON_WINDOWED,
-    WINDOWED,
-    UNKNOWN
```

Comment:
Similarly, if we actually have a plan, we know whether or not it's a windowed source.